lib/session_encryptor.rb

```diff
@@ -39,10 +40,17 @@ class SensitiveValueError < StandardError; end
   def load(value)
     return LegacySessionEncryptor.new.load(value) if should_use_legacy_encryptor_for_read?(value)
 
-    _v2, ciphertext = value.split(':')
+    payload = MessagePack.unpack(value)
+    ciphertext = payload['t']
+    compressed = payload['c']
```

Comment:
what if we we made string constants for these keys? so they're still small in the payload but then we get some readability?

```suggestion
    ciphertext = payload[CIPHERTEXT]
    compressed = payload[COMPRESSED]
```

Reply:
done!